ui/app/AppLayouts/Communities/panels/SharedAddressesPermissionsPanel.qml

```diff
@@ -143,6 +145,20 @@ Rectangle {
                     font.weight: Font.Medium
                     text: qsTr("Permissions")
                 }
+                Item { Layout.fillWidth: true }
+                RowLayout {
```

Comment:
According to design it should have more margin in the right side:
- Design:

![Screenshot 2024-03-26 at 13 18 46](https://github.com/status-im/status-desktop/assets/97019400/e6c74b4d-cdf0-4bec-b071-5ae5889d438b)

- App:

![Screenshot 2024-03-26 at 13 20 29](https://github.com/status-im/status-desktop/assets/97019400/0bae112c-266d-4d48-b91f-ec02c712af6a)

Reply:
@noeliaSD fixed :)

![image](https://github.com/status-im/status-desktop/assets/5377645/1a304060-11b0-41c9-abf7-fa28c33b7e01)